app/v1/controllers/requestParsers/validators/validations/ConsolidatedExpensesValidation.scala

```diff
@@ -17,45 +17,24 @@
 package v1.controllers.requestParsers.validators.validations
 
 import v1.models.errors.{MtdError, RuleBothExpensesSuppliedError}
-import v1.models.request.amendForeignProperty._
-import v1.models.request.createForeignProperty._
+import v1.models.request.common.foreignFhlEea
```

Comment:
Change to `import v1.models.request.common.foreignFhlEea._` and `import v1.models.request.common.foreignPropertyEntry._` if you're using stuff inside these packages